cpp/include/raft/sparse/cusparse_wrappers.h

```diff
@@ -26,6 +26,24 @@
   case err:                       \
     return #err;
 
+//Notes:
+//(1.) CUDA_VER_SELECT aggregates all the CUDA version selection logic;
+//(2.) to enforce a lower version,
+//
+//`#define CUDA_ENFORCE_LOWER
+// #include <raft/sparse/cusparse_wrappers.h>`
+//
+// (i.e., before including this header)
+//
+
+#define CUDA_VER_SELECT (CUDART_VERSION >= 10100)
```

Comment:
Maybe a name other than CUDA_VER_SELECT, since SELECT doesn't make it clear we mean "high version," like CUDA_VER_RECENT or CUDA_VER_POST101?

Reply:
So, the meaning meant to be conveyed was a "way to detect the active cuda toolkit version". It might not necessarily be _recent_. A pre-10.1 user should still be able to compile/run these. Just with a different code-path. (I know we don't support 10.0 or below anymore, but it's good to still have a plan B, plus it helps w/ debugging)

Reply:
How about `CUDA_VER_10_1_UP` (`POST101` may suggest 10.1 is not included, when in fact it is).

Reply:
Sounds good to me. The underscores are definitely clearer than my POST101 suggestion!

Reply:
Done.